nixos/tests/certmgr.nix

```diff
@@ -20,9 +20,9 @@ let
           owner = "nginx";
           path = "/var/ssl/${host}-ca.pem";
         };
-        label = "www_ca";
-        profile = "three-month";
+        profile = "default";
         remote = "localhost:8888";
+        auth_key_file = "/var/ssl/apitoken";
```

Comment:
I think in the current version of `certmgr` there's no way to configure it to not use auth. It seemed like a better test to use auth anyway so I've added it here.